notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+    "\n",
+    "```bash\n",
+    "multitail -CS log4jnew -du -P a -s 2 -sn 1,3 \\\n",
+    "    -e \"WARN\" -f coordinator-overlord.log \\\n",
+    "    -ev \"INFO\" -f broker.log \\\n",
+    "    -ev \"INFO\" -f middlemanager.log \\\n",
+    "    -ev \"INFO\" -f historical.log\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1023e5a3-cedc-435f-a448-30f8c3ae1535",
+   "metadata": {},
+   "source": [
+    "# Clean up\n",
+    "\n",
+    "Run this command to stop Druid.\n",
+    "\n",
+    "```bash\n",
+    "kill $(ps -ef | grep 'supervise' | awk 'NF{print $2}' | head -n 1)\n",
+    "```\n",
+    "\n",
+    "Remove the `learn-druid-local` folder from your home folder in the usual way."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57d5921b-636e-4a85-ae4d-dc0239bfef0b",
+   "metadata": {},
+   "source": [
+    "## Learn more\n",
+    "\n",
+    "You've seen how Log4J is used by Apache Druid, and about the default configurations.\n",
+    "\n",
+    "For more information:\n",
+    "\n",
+    "* Watch [Druid optimizations for scaling customer facing analytics at Conviva](https://youtu.be/zkHXr-3GFJw?t=746) by Amir Youssefi and Pawas Ranjan from Conviva that describes how useful this information can be to tuning Druid clusters.\n",
+    "* Read more about Druid [logging](https://druid.apache.org/docs/latest/configuration/logging.html) in the official documentation.\n",
+    "* See more ways to use and run `multitail` on the [official site](https://www.vanheusden.com/multitail/index.html).\n",
+    "* Read more about log4j in [Apache Logging Services](https://logging.apache.org/) documentation.\n",
+    "* Read your options for `filePattern`s by refering to the [simple date format](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html) documentation.\n",
+    "* For different `PatternLayout`s refer to the [pattern layout](https://logging.apache.org/log4j/2.x/manual/layouts.html#pattern-layout) documentation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c081295-750e-41a3-9693-7fe50af401fa",
+   "metadata": {},
+   "source": [
+    "Notice that the Logger Name shows a fully-qualified class name. Here are some examples of searches you may want to conduct on your logs to deepen your knowledge of what your instance is doing.\n",
+    "\n",
+    "| Log | Search Term |\n",
+    "|---|---|\n",
+    "| Any | __NodeRoleWatcher__<br>Across all the processes, watch as they detect changes in the processes that are running in the cluster, and see what they do about it! |\n",
+    "| Any | __org.apache.druid.initialization.Initialization__<br>These messages are all about the process starting up. It can be interesting to see what exactly each one does – and if it runs into issues. |\n",
```

Comment:
```suggestion
    "| Any | __org.apache.druid.initialization.Initialization__<br>These messages are all about the process starting up. It can be interesting to see what exactly each one does and if it runs into issues. |\n",
```